applications/nrf5340_audio/src/audio/audio_sync_timer.c

```diff
@@ -79,6 +89,79 @@ static void event_handler(nrf_timer_event_t event_type, void *ctx)
 {
 }
 
+static int audio_sync_timer_sync_led_init(void)
+{
+	int ret;
+
+	/* Connect the IRQ handler for timer 1 */
+	if (AUDIO_SYNC_TIMER_INSTANCE == 1) {
+		IRQ_CONNECT(DT_IRQN(DT_NODELABEL(timer1)), AUDIO_SYNC_TIMER_IRQ_PRIORITY, nrfx_isr,
+			    nrfx_timer_1_irq_handler, 0);
+	} else {
+		LOG_WRN("Invalid audio sync timer instance");
+		return -ECANCELED;
+	}
+
+	if (nrfx_gpiote_is_init()) {
+		LOG_DBG("GPIOTE is already initiated");
+	} else {
+		nrfx_gpiote_init(0);
+		LOG_DBG("GPIOTE initialized");
+	}
+
+	/* GPIOTE out configs for sync LED */
+	nrfx_gpiote_out_config_t const gpiote_cfg = NRFX_GPIOTE_CONFIG_OUT_TASK_HIGH;
+
+	ret = nrfx_gpiote_out_init(SYNC_LED, &gpiote_cfg);
+	if (ret - NRFX_ERROR_BASE_NUM) {
+		LOG_ERR("nrfx GPIOTE init error - Return value: %d", ret);
+		return ret;
+	}
+
+	nrfx_gpiote_out_task_enable(SYNC_LED);
+
+	/* Initializing SET task for GPIOTE (sync LED) */
+	ret = nrfx_dppi_channel_alloc(&dppi_channel_gpiote_set);
+	if (ret - NRFX_ERROR_BASE_NUM) {
+		LOG_ERR("nrfx DPPI channel alloc error (GPIOTE SET task) - Return value: %d", ret);
+		return ret;
+	}
+
+	nrf_gpiote_task_t gpiote_set_task = nrfx_gpiote_set_task_get(SYNC_LED);
+	nrf_timer_event_t sync_led_on_cmpr_evt =
+		nrf_timer_compare_event_get(AUDIO_SYNC_TIMER_SYNC_LED_ON_CMPR_CHANNEL);
```

Comment:
It is possible to use this function for NRF_TIMER_TASK_CAPTURE0 as well?

Reply:
Yes, one can use "nrf_timer_capture_task_get", added a suggestion where this is implemented.